Confus/Confus/Game.cpp

```diff
@@ -246,8 +246,7 @@ namespace Confus
     void Game::fixedUpdate()
     {
 		m_MazeGenerator.fixedUpdate();
-		m_PhysicsWorld.stepSimulation(static_cast<float>(FixedUpdateInterval));
-		m_PlayerHandler.fixedUpdate();
+		//m_PhysicsWorld.stepSimulation(static_cast<float>(FixedUpdateInterval));
```

Comment:
I know, but can you remove the commented line as well, because it will conflict with my change of removing the line entirely